set.mm

```diff
@@ -215793,6 +215905,28 @@ everywhere defined internal operation (see ~ mndcl ), whose operation is
       PMN $.
   $}
 
+  ${
+    hashelne0d.1 $e |- ( ph -> B e. A ) $.
+    hashelne0d.2 $e |- ( ph -> A e. V ) $.
+    $( A set with an element has nonzero size.  (Contributed by Rohan Ridenour,
+       3-Aug-2023.) $)
+    hashelne0d $p |- ( ph -> -. ( # ` A ) = 0 ) $=
```

Comment:
Theorem should be placed near after ~hasheq0.

Reply:
Placed it after `hash0`, I think it's a bit more logical that way.